samples/kind-lb/setupkind.sh

```diff
@@ -85,9 +79,6 @@ done
 # connect it to the docker network where kind nodes are running on
 # which normally will be called kind
 FEATURES=$(cat << EOF
-featureGates:
-  MixedProtocolLBService: true
-  GRPCContainerProbe: true
```

Comment:
accroding https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/, we didn't need this with 1.28+